roles/rsyslog/roles/input_roles/ovirt/defaults/main.yaml

```diff
@@ -87,10 +87,10 @@ rsyslog_conf_ovirt_formatting:
               property(name="TIMESTAMP" dateFormat="rfc3339")
           }
 
-          input (type="imtcp" port="{{ rsyslog_read_collectd_port|d(44514) }}")
+          input (name="ovirt_imtcp" type="imtcp" port="{{ rsyslog_read_collectd_port|d(44514) }}")
```

Comment:
@nhosoi 
This is failing oVirt configuration
error during parsing file /etc/rsyslog.d/20-ovirt_formatting.conf, on or before line 30: parameter 'name' not known -- typo in config file? [v8.24.0-41.el7_7.2 try http://www.rsyslog.com/e/2207 ]

I believe you can only add name to "action".

Reply:
Ahhh, got it.  After writing the following note, I noticed the failed line is 30, which is not for imtcp...  Could you please find out what the line 30 has?   The parameter `name` was added for the stats.  So, we'd like to keep them as much as possible. :)

==========================================================================================
Very strange...  I have parameter `name` in basics role, as well.  And I have no problem to start rsyslog [1]. 
```
module(load="imtcp")
........
input(
  name="basics_imtcp"
  type="imtcp"
  port="514"
)
```
Here's the doc of imtcp input parameters, where `name` is supported.
https://www.rsyslog.com/doc/v8-stable/configuration/modules/imtcp.html#input-parameters
Please note that I'm running the same version of rsyslog...
```
# rpm -q rsyslog
rsyslog-8.24.0-41.el7_7.2.x86_64
```
[1]
```
# systemctl status rsyslog -l
● rsyslog.service - System Logging Service
   Loaded: loaded (/usr/lib/systemd/system/rsyslog.service; enabled; vendor preset: enabled)
   Active: active (running) since Sun 2020-01-26 10:06:00 PST; 2min 27s ago
     Docs: man:rsyslogd(8)
           http://www.rsyslog.com/doc/
 Main PID: 19367 (rsyslogd)
    Tasks: 9
   CGroup: /system.slice/rsyslog.service
           └─19367 /usr/sbin/rsyslogd -n

Jan 26 10:06:00 momo7.localdomain systemd[1]: Stopped System Logging Service.
Jan 26 10:06:00 momo7.localdomain systemd[1]: Starting System Logging Service...
Jan 26 10:06:00 momo7.localdomain rsyslogd[19367]:  [origin software="rsyslogd" swVersion="8.24.0-41.el7_7.2" x-pid="19367" x-info="http://www.rsyslog.com"] start
Jan 26 10:06:00 momo7.localdomain systemd[1]: Started System Logging Service.
```